### ISSUE TITLE
Unable to Build this Project 

### ISSUE THREAD

**Issue by glacier6327** [groupon/Selenium-Grid-Extras#365]
I am having the issue with the https download from my windows 10 boxes and also need to get the fix that was put in with the last release. 

I can get these if I can build the project locally, but I am getting the following error when I attempt to build.

Please help...

rmarchionne@CVG-000211M-173 MINGW64 /c/Development/git/Selenium-Grid-Extras (master)
$ mvn clean install -U
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] SeleniumGridExtras
[INFO] Selenium Grid Extras
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building SeleniumGridExtras 1.12.15-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master/org/apache/ant/ant/maven-metadata.xml
Downloading: http://repo1.maven.org/maven2/org/apache/ant/ant/maven-metadata.xml
Downloading: https://repo.maven.apache.org/maven2/org/apache/ant/ant/maven-metadata.xml
Downloaded: http://repo1.maven.org/maven2/org/apache/ant/ant/maven-metadata.xml (886 B at 2.0 kB/s)
Downloaded: https://repo.maven.apache.org/maven2/org/apache/ant/ant/maven-metadata.xml (886 B at 1.5 kB/s)
[WARNING] Could not transfer metadata org.apache.ant:ant/maven-metadata.xml from/to xuggle repo (https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master): sun.secur
ity.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested
target
Downloading: http://repo1.maven.org/maven2/xuggle/xuggle-xuggler/5.4/xuggle-xuggler-5.4.pom
Downloading: https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master/xuggle/xuggle-xuggler/5.4/xuggle-xuggler-5.4.pom
Downloading: https://repo.maven.apache.org/maven2/xuggle/xuggle-xuggler/5.4/xuggle-xuggler-5.4.pom
Downloading: http://repo1.maven.org/maven2/xuggle/xuggle-xuggler/5.2/xuggle-xuggler-5.2.pom
Downloading: https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master/xuggle/xuggle-xuggler/5.2/xuggle-xuggler-5.2.pom
Downloading: https://repo.maven.apache.org/maven2/xuggle/xuggle-xuggler/5.2/xuggle-xuggler-5.2.pom
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] SeleniumGridExtras ................................. FAILURE [  2.346 s]
[INFO] Selenium Grid Extras ............................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.468 s
[INFO] Finished at: 2017-09-12T10:12:34-04:00
[INFO] Final Memory: 13M/210M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project SeleniumGridExtras: Could not resolve dependencies for project com.groupon.selenium-grid-extras:SeleniumGridExtras:jar:1.12.15-SNAPS
HOT: Failed to collect dependencies at xuggle:xuggle-xuggler:jar:5.4: Failed to read artifact descriptor for xuggle:xuggle-xuggler:jar:5.4: Could not transfer artifact xuggle
:xuggle-xuggler:pom:5.4 from/to xuggle repo (https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master): sun.security.validator.ValidatorException: PKIX path building
 failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

**Comment by glacier6327** [comment]
Shawn McCarthy ??? Any help? This is your repo... Thanks really in need of some help with just building in Maven,,

**Comment by smccarthy** [comment]
@glacier6327 Do you still get this error with the latest from master ? I'll try and dig into it today.

**Comment by glacier6327** [comment]
@smccarthy  Yes sir... seem to be an issue with xuggle... which looks like is also you repo...  Really could use some help.. 

Failed to execute goal on project SeleniumGridExtras: Could not resolve dependencies for project com.groupon.selenium-grid-extras:SeleniumGridExtras:jar:1.12.16-SNAPS
HOT: Failed to collect dependencies at xuggle:xuggle-xuggler:jar:5.4: Failed to read artifact descriptor for xuggle:xuggle-xuggler:jar:5.4: Could not transfer artifact xuggle
:xuggle-xuggler:pom:5.4 from/to xuggle repo (https://raw.github.com/smccarthy/xuggle-xuggler-dependency/master): sun.security.validator.ValidatorException: PKIX path building
 failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]

**Comment by smccarthy** [comment]
@glacier6327 It looks like a problem with the https for you... I have seen similar issues with people not being about to get chromedriver with selenium-grid-extras. Can you remove the `s` from the `https` and try it?

Here is the line you need to change :
https://github.com/groupon/Selenium-Grid-Extras/blob/master/SeleniumGridExtras/pom.xml#L17

**Comment by glacier6327** [comment]
That does not seem to have any impact... unfortuanely. I even tried the Maven commands to ignore SSL errors, but I still see them.. frustrating...  

Could not resolve dependencies for project com.groupon.selenium-grid-extras:SeleniumGridExtras:jar:1.12.16-SNAPS
HOT: Failed to collect dependencies at xuggle:xuggle-xuggler:jar:5.4: Failed to read artifact descriptor for xuggle:xuggle-xuggler:jar:5.4: Could not transfer artifact xuggle
:xuggle-xuggler:pom:5.4 from/to xuggle repo (http://raw.github.com/smccarthy/xuggle-xuggler-dependency/master): sun.security.validator.ValidatorException: PKIX path building
failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target -> [Help 1]
